include/wx/translation.h

```diff
@@ -45,30 +45,19 @@ using wxTranslationsHashMap = std::unordered_map<wxString, wxString>;
 // --keyword="_" --keyword="wxPLURAL:1,2" options
 // to extract the strings from the sources)
 #ifndef WXINTL_NO_GETTEXT_MACRO
-#ifndef wxNO_IMPLICIT_WXSTRING_ENCODING
-    #define _(s)                               wxGetTranslation((s))
-#else
-    #define _(s)                               wxGetTranslation(wxASCII_STR(s))
-#endif
+    #define _(s)                               wx_Wrapper((s))
```

Comment:
Pretty minor, but could we call it something more readable, e.g.

```suggestion
    #define _(s)                               wxUnderscoreWrapper((s))
```

?